ChatSX/constants/Numbers.ts

```diff
@@ -0,0 +1,10 @@
+export const TWO = 2;
+export const THREE = 3;
+export const FOUR = 4;
+export const SIX = 6;
+export const EIGHT = 8;
+export const NINE = 9;
+export const FIFTEEN = 15;
+export const TWENTY_ONE = 21;
+export const TWENTY_SEVEN = 27;
+export const TWENTY_EIGHT = 28;
```

Comment:
plus, these are not enums, they are just constants, use proper enums in js/ts. Follow this: https://www.w3schools.com/typescript/typescript_enums.php